config/config.go

```diff
@@ -84,6 +84,24 @@ func (hc *HipChat) Merge(other *HipChat) {
 	setIfNotEmpty(&hc.APIKey, other.APIKey)
 }
 
+// Valid returns true if the HipChat config is ready to be used for
+// HipChat notifications.
+func (hc *HipChat) Valid() bool {
```

Comment:
Should it maybe post a test notification so that it can see if the key is valid?